agagd/templates/agagd_core/chapter.html

```diff
@@ -46,7 +46,7 @@ <h2> Chapter: {{ chapter.code }} </h2>
       <dt> Website: </dt> <dd> <a href="{{chapter.url}}">{{ chapter.url }}</a> </dd>
   {% endif %}
 </dl>
-
+<p></p>
```

Comment:
Can you do something different here for your formatting? I am not sure we should be using `<p>` and `</p>` tags here if we are looking for a line break. CSS or Inline style maybe?

Reply:
I could add a double break line <br> in its place. Wanted just a large spacing there.